src/jdk.jlink/share/classes/jdk/tools/jlink/resources/jlink.properties

```diff
@@ -111,10 +111,16 @@ main.extended.help.footer=\
 \              used, one pattern per line\n\
 \n\
 
+main.capability.runtime=Capabilities: {0}run-time-image
 
 error.prefix=Error:
 warn.prefix=Warning:
 
+err.runtime.link.not.linkable.runtime=The current run-time image does not support run-time linking.
```

Comment:
Please remove the trailing periods from the new messages. The existing messages don’t use them, and JDK command-line tools in general don’t use them.

Reply:
Sure.